src/Counter/ButtonList.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+function ButtonList(onClickButton) {
+    return <div>
+        <button className="btn btn-primary" name="inc" onClick={() => onClickButton.onClickHandler(1)}>Increment</button>
+        <button name="dec" className="btn btn-primary" onClick={() => onClickButton.onClickHandler(-1)}>Decrement</button>
+    </div>
+}
+export default ButtonList
```

Comment:
let's get rid of this waring by adding space 
<img width="355" alt="image" src="https://github.com/GaKhalatyan86/ReactLessons/assets/36797396/e5f3a21c-eac3-429e-87d2-29ebff53ff77">

Reply:
ButtonList component removed